common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1378,17 +1388,47 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(1000)
 
-  /**
-   * @return This configuration is a guidance for load-aware slot allocation algorithm. This value
-   *         is control how many disk groups will be created.
-   */
-  def diskGroups(conf: CelebornConf): Int = {
-    conf.getInt("rss.disk.groups", 5)
-  }
+  val DISK_GROUP: ConfigEntry[Int] =
+    buildConf("celeborn.disk.group")
+      .withAlternative("rss.disk.groups")
+      .categories("master")
+      .doc("This configuration is a guidance for load-aware slot allocation algorithm. " +
+        "This value is control how many disk groups will be created.")
+      .version("0.2.0")
+      .internal
+      .intConf
+      .createWithDefault(5)
 
-  def diskGroupGradient(conf: CelebornConf): Double = {
-    conf.getDouble("rss.disk.group.gradient", 0.1)
-  }
+  val DISK_GROUP_GRADIENT: ConfigEntry[Double] =
+    buildConf("celeborn.disk.group.gradient")
+      .withAlternative("rss.disk.groups.gradient")
+      .categories("master")
+      .doc("This value means how many more workload will be placed into a faster disk group " +
+        "than a slower group.")
+      .version("0.2.0")
+      .internal
+      .doubleConf
+      .createWithDefault(0.1)
+
+  val MASTER_OFFER_SLOTS_EXTRA_SIZE: ConfigEntry[Int] =
+    buildConf("celeborn.master.offerSlots.extraSize")
+      .withAlternative("rss.offer.slots.extra.size")
+      .categories("master")
+      .version("0.2.0")
+      .doc("")
```

Comment:
ping @waitinfuture What should I write here?

Reply:
As we discussed, let's use the following names
```
celeborn.slots.assign.policy
celeborn.slots.assign.loadAware.numDiskGroups
celeborn.slots.assign.loadAware.diskGroupGradient
celeborn.slots.assign.extraSlots
```